src/screens/how-to-help/raise-awareness.css

```diff
@@ -0,0 +1,27 @@
+.link-copied-toast[data-state='open'] {
+  animation: slideUp 150ms ease-out forwards;
+}
+
+.link-copied-toast[data-state='closed'] {
+  animation: slideDown 150ms ease-in forwards;
+}
+
+@keyframes slideUp {
+  from {
+    @apply opacity-0;
+    transform: translateY(0);
+  }
+  to {
+    @apply opacity-100 -translate-y-4;
+  }
+}
+
+@keyframes slideDown {
+  from {
+    @apply opacity-100 -translate-y-4;
+  }
+  to {
+    @apply opacity-0;
+    transform: translateY(0);
+  }
+}
```

Comment:
Unfortunately, it seems that Radix doesn't play well with transitions, so you need to use CSS animations for enter/exit animations. I tried using Framer Motion to avoid writing custom CSS, but Radix _also_ doesn't play well with Framer Motion's `AnimatePresence` component, so I couldn't animate during the toast's unmount.

Reply:
I did not run into issues with animating the Accordion component from Radix, so this sounds like a bug maybe? A CSS animation is probably fine, though.